requirements/base.txt

```diff
@@ -275,7 +288,7 @@ path==16.16.0
     # via edx-i18n-tools
 pbr==6.1.1
     # via stevedore
-phonenumbers==8.13.55
+phonenumbers==9.0.0
```

Comment:
could you check if this change breaks something ?

Reply:
Seems that in python nothing. Anyway, we need to keep monitoring in case.
https://github.com/daviddrysdale/python-phonenumbers/blob/dev/python/HISTORY.md?plain=1#L15-L19